.github/workflows/update-bpmn-visualization-version.yml

```diff
@@ -15,33 +15,33 @@ jobs:
       VERSION: ${{ github.event.client_payload.version || github.event.inputs.version }}
     steps:
       - uses: actions/checkout@v3
-      - name: Get old bpmn-visualization version
+      - name: Get the old version of the bpmn-visualization TypeScript library
         id: lookupOldBPMNVisuVersion
         uses: mikefarah/yq@v4.30.5
         with:
-          cmd: yq '.dependencies[] | select(.name == "bpmn-visualization") | .version' inst/htmlwidgets/bpmnVisualization.yaml
+          cmd: yq '.dependencies[] | select(.name == "bpmn-visualization") | .version' inst/htmlwidgets/bpmnVisualizationR.yaml
       - run: echo "OLD_VERSION=${{ steps.lookupOldBPMNVisuVersion.outputs.result }}" >> $GITHUB_ENV
-      - name: Delete old bpmn-visualization js lib file
+      - name: Delete the old js file of the bpmn-visualization TypeScript library
         run: rm inst/htmlwidgets/lib/bpmn-visualization/bpmn-visualization.min.js
-      - name: Download bpmn-visualization ${{ env.VERSION }}
+      - name: Download the ${{ env.VERSION }} js file of the bpmn-visualization TypeScript library
         uses: carlosperate/download-file-action@v2
         with:
           file-url: 'https://cdn.jsdelivr.net/npm/bpmn-visualization@${{ env.VERSION }}/dist/bpmn-visualization.min.js'
           location: 'inst/htmlwidgets/lib/bpmn-visualization'
-      - name: Update bpmn-visualization version in YAML file
+      - name: Update the version of the bpmn-visualization TypeScript library in the YAML file
         run: |
-          sed -i -E 's/version: ".*"/version: "${{ env.VERSION }}"/' inst/htmlwidgets/bpmnVisualization.yaml
+          sed -i -E 's/version: ".*"/version: "${{ env.VERSION }}"/' inst/htmlwidgets/bpmnVisualizationR.yaml
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v4.2.3
         with:
           token: ${{ secrets.GH_RELEASE_TOKEN }}
-          commit-message: "[INFRA] Bump bpmn-visualization JS lib from ${{ env.OLD_VERSION }} to ${{ env.VERSION }}"
+          commit-message: "[INFRA] Bump bpmn-visualization TypeScript library from ${{ env.OLD_VERSION }} to ${{ env.VERSION }}"
           committer: "process-analytics-bot <62586190+process-analytics-bot@users.noreply.github.com>"
           author: "process-analytics-bot <62586190+process-analytics-bot@users.noreply.github.com>"
           branch: "infra/bump_bpmn_visualization_from_${{ env.OLD_VERSION }}_to_${{ env.VERSION }}"
           delete-branch: true
           base: "main"
-          title: "[INFRA] Bump bpmn-visualization JS lib from ${{ env.OLD_VERSION }} to ${{ env.VERSION }}"
+          title: "[INFRA] Bump bpmn-visualization TypeScript library from ${{ env.OLD_VERSION }} to ${{ env.VERSION }}"
```

Comment:
Same comment for commit-message and title as in https://github.com/process-analytics/bpmn-visualization-examples/pull/436
Here we can keep bpmn-visualization only (and remove JS lib) as we clearly differentiate the R package (bpmnVisualizationR) from the TS lib (bpmn-visualization).